tools/ansible/qemu_cloud_start.yml

```diff
@@ -104,5 +104,8 @@
           now to manage the cloud you can do:
 
           clusterssh {{ linklocals | join(' ') }}
+          
+          to access to node you can do:
```

Comment:
This ends up printing:
```
to access to node you can do:
ssh -o StrictHostKeyChecking=no -o"HostkeyAlgorithms +ssh-rsa" fe80::5000:ff:feab:c000%lm_cloudC fe80::5000:ff:feab:c001%lm_cloudC fe80::5000:ff:feab:c002%lm_cloudA fe80::5000:ff:feab:c003%lm_cloudA fe80::5000:ff:feab:c004%lm_cloudA fe80::5000:ff:feab:c005%lm_cloudB fe80::5000:ff:feab:c006%lm_cloudB fe80::5000:ff:feab:c007%lm_cloudB fe80::5000:ff:feab:c008%lm_cloudB fe80::5000:ff:feab:c009%lm_cloudD fe80::5000:ff:feab:c010%lm_cloudD fe80::5000:ff:feab:c011%lm_cloudD
```

which has two issues:
- user root should be used "root@fe80::5000:ff:feab:c000%lm_cloudC"
- ssh takes only one host as parameter not multiple hosts.

Reply:
Also, we could print  -o options for clusterssh too